slash-commands/poll/index.ts

```diff
@@ -34,14 +35,14 @@ const poll = async (interaction) => {
         embeds: [
             {
                 title: title,
-                color: randomColor(),
+                color: randomColor() as ColorResolvable,
```

Comment:
Laita randomColor() palauttamaan colorResolvable sen sijaan että castaat sen vasta täällä. En oo testannu mut ton alemman koodin pitäis toimii

```ts
export const randomColor = (): ColorResolvable => {
    let color = "";
    for (let i = 0; i < 3; i++) {
        color += _.random(0, 255).toString(16);
    }
    return `#${color}`;
};
```